MapboxNavigation/Constants.swift

```diff
@@ -16,6 +16,14 @@ public let MBRouteLineWidthByZoomLevel: [Int: NSExpression] = [
     22: NSExpression(forConstantValue: 28)
 ]
 
+public let MLNRouteLineWidthByZoomLevel: [Int: Double] = [
+	10: 8,
+	13: 9,
+	16: 11,
+	19: 22,
+	22: 28
+]
+
```

Comment:
Please stick to the established indentation style.

```suggestion
public let MLNRouteLineWidthByZoomLevel: [Int: Double] = [
    10: 8,
    13: 9,
    16: 11,
    19: 22,
    22: 28
]

```